tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>Is there a reason for using tf.compat.v1 instead of the TF2 version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>ah, it is because at the time I wrote the notebook, the APIs do not work with <code>from_keras_model </code>in TF 2.x. We talked about it and considered it to be trivial?</p><p>However, I just did a quick test using the fix from this <a href="https://github.com/tensorflow/model-optimization/pull/422" target="_blank">PR</a> (also resolves a failed test in the tensorflow/models <a href="https://github.com/tensorflow/models/pull/8604" target="_blank">PR</a>.), and it is currently working fine. Will test more later.</p>

Reply:
<p>Is there a tensorflow-model-optimization-dev.xxx package for me to install at the beginning of the tutorial that includes this PR mentioned above? Or, should I stick with '!&nbsp;pip&nbsp;install&nbsp;--upgrade&nbsp;tensorflow-model-optimization', which fails the <code>from_keras_model</code> function.</p>

Reply:
This is fine for now - I'll create a dev package sometime soon though for testing.